x-pack/plugins/maps/public/classes/sources/kibana_regionmap_source/kibana_regionmap_layer_wizard.tsx

```diff
@@ -12,13 +12,12 @@ import { KibanaRegionmapSource, sourceTitle } from './kibana_regionmap_source';
 import { VectorLayer } from '../../layers/vector_layer/vector_layer';
 // @ts-ignore
 import { CreateSourceEditor } from './create_source_editor';
-// @ts-ignore
 import { getKibanaRegionList } from '../../../meta';
 
 export const kibanaRegionMapLayerWizardConfig: LayerWizard = {
-  checkVisibility: () => {
+  checkVisibility: async () => {
```

Comment:
yes, because the interface is Promise<boolean> in https://github.com/elastic/kibana/blob/master/x-pack/plugins/maps/public/classes/layers/layer_wizard_registry.ts#L23